admin/src/main/resources/io/buoyant/admin/js/template/delegator.handlebars

```diff
@@ -15,7 +15,7 @@
     Dtab
   </h4>
   <div id="dtab"></div>
-  <div class="namerd-dtab-warning hide">Note: the above Dentries are from namerd and can't be edited.</div>
+  <div class="namerd-dtab-warning hide">Note: the above Dentries are from an external source and can't be edited.</div>
```

Comment:
(I think you'll need to recompile the templates in order for this change to take effect)